### PR TITLE
[ADD] sms marketing: added essentials/getting started docs

### PR DESCRIPTION
**Pull request by ksc-odoo**
Task ID: [2942125](https://www.odoo.com/web#id=2942125&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hi @mivu-odoo 

I added some basic, "essential," documentation to the SMS Marketing doc.

Once I get your approval/review, I'll pass it along to Zac.

Thanks!

**Comment by mivu-odoo** [comment]
Hi @ksc-odoo!

This PR wasn't passing the `ci/documentation` code check b/c of some build errors.

As of commit [d542657](https://github.com/odoo/documentation/commit/d5426579f41ab6bfac8a8cacdc7c74da670c42e5)

- Renamed the RST files to replace the hyphens with underscores in `contacts-blacklist.rst` and `sms-integrations.rst`
- Renamed `first-campaign.rst` to `first_sms_campaign.rst` to be more specific b/c I know PR #2552 is going to add a file named `first_campaign.rst` for Marketing Automation user docs, and I thought it would be confusing to have two files in different places with the same name
- Fixed the RST file names referenced in `essentials.rst` to match the new RST file names (the ones with the underscores)
- Renamed the image folders to replace the hyphens with underscores (so they match the new RST file names)
- Fixed the image folder name references in `first_sms_campaign.rst`, `sms_integrations.rst`, and `contacts_blacklist.rst`
- Fixed the indentation of `sms_marketing/essentials` and `sms_marketing/pricing` in `sms_marketing.rst` to align with the first colon of `:titlesonly:`

Despite all these changes, the PR still won't pass the code check b/c there's one error I haven't quite figured out how to crack. 

When I run `make html`, I get an error for `first_sms_campaign.rst:140: WARNING: unknown document: pricing_and_faq`. I tried several different iterations like including more of the file path, but I would still get the same build error. Will probably have to loop in ZST to fix.

**Comment by samueljlieber** [comment]
Hi @mivu-odoo and @ksc-odoo 👋

I fixed what was causing the build error here! It was due to an incorrect referenced doc path on line 140 of `first_sms_campaign.rst` 🙂

The path was: 
`pricing_and_faq`

The referenced doc path needs to be [relative to the document](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#contributing-relative-links) you are linking from, so the correct path looks like this:
`../pricing/pricing_and_faq`

**Comment by ksc-odoo** [comment]
@StraubCreative Ready for your review!

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

These docs need another round of proofreading and revisions.

Lots of 2nd-person _you/your_ language, some screenshots are too big, unnecessary, or not named properly, etc...the usual suspects 😉 

Please submit updated content to @samueljlieber he'll handle the git flow for you.

**Comment by samueljlieber** [comment]
Hi @ksc-odoo, in b4acc53 I implemented your changes per @StraubCreative [suggestions](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1284380372):
- FYI renamed the following files:
    - `first_sms_campaign.rst` to `sms_essentials.rst`
    - `sms_integrations.rst` to `sms_campaigns.rst`
    - `contacts_blacklists.rst` to `mailing_lists_blacklists.rst`
- `sms_essentials.rst` lines `29`-`35` & lines `136`-`142` I formatted as a list rather than sentences because they are referring to options that can be selected from a dropdown, it made sense to me for the options to be displayed in a list.

I think your changes to these docs look really good 👍 I made a few small RST adjustments, @ksc-odoo can you please confirm these changes with a review before I pass to @StraubCreative? 🙏

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo, in [b4acc53](https://github.com/odoo/documentation/commit/b4acc5309e3bb6ad12dffbd3c88b9923a2ee9225) I implemented your changes per @StraubCreative [suggestions](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1284380372):
> 
> * FYI renamed the following files:
>   
>   * `first_sms_campaign.rst` to `sms_essentials.rst`
>   * `sms_integrations.rst` to `sms_campaigns.rst`
>   * `contacts_blacklists.rst` to `mailing_lists_blacklists.rst`
> * `sms_essentials.rst` lines `29`-`35` & lines `136`-`142` I formatted as a list rather than sentences because they are referring to options that can be selected from a dropdown, it made sense to me for the options to be displayed in a list.
> 
> I think your changes to these docs look really good 👍 I made a few small RST adjustments, @ksc-odoo can you please confirm these changes with a review before I pass to @StraubCreative? 🙏

Hey @samueljlieber - Feel free to pass it along to @StraubCreative 👍

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+.. note::
+   Credits must be purchased from Odoo in order to take advantage of the app (and its features).
+   SMS messages will not be sent without credits.
+
+.. seealso::
+   - :doc:`../pricing/pricing_and_faq`
+
+How to track links used in SMS messages
+---------------------------------------
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:107]: Note for all of the headings that start with `How to...`
We can just remove that part and stick with the action that the heading conveys for the section. Please apply to all headings in this and other docs.

```suggestion
Track links used in SMS messages
--------------------------------
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+==========================
+
+Under the :guilabel:`Settings` tab of the SMS template, there is an option to :guilabel:`Include
+opt-out link`. If activated, the recipient is able to unsubscribe from the mailing list, thus
+avoiding all future mailings.
+
+An employee can be designated as the :guilabel:`Responsible` in the :guilabel:`Tracking` section of
+the :guilabel:`Settings` tab, as well.
+
+.. image:: sms_essentials/sms-settings-tab.png
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:127]: Too wide. Doesn't appear to be in HD.

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+   :align: center
+   :alt: SMS Settings tab.
+
+How to send SMS messages
+========================
+
+Once a mailing is created, choose when Odoo should deliver the message from the following options:
+
+- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
+  isn't excessive.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:137]: Point of clarity: what does `excessive` mean here? How is this defined?

Consider the following suggestion, instead:
```suggestion
- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
  is highly refined or in cases that involve fast approaching deadlines, such as a flash sale.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  limited-time offer, or to help plan a company's content strategy in advance.
+- :guilabel:`Test`: allows for an SMS to be sent to one or multiple numbers for test purposes.
+  Remember to use a comma between phone numbers, if multiple numbers are used as recipients.
+
+Visualize reports
+=================
+
+On the :guilabel:`Reporting` page (accessible via the :guilabel:`Reporting` option in the header
+menu), there are options to apply filters, measures, and view metrics in a number of different
+layouts (:guilabel:`Graph`, :guilabel:`List`, and :guilabel:`Cohort`).
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:149]: I think in this case, even though it's solitary, the second tag is actually a `menuselection`. As well, I would indicate`filters` and `measures` as `guilabels`. Can remove the parenthesis in the last sentence too and just write out what those views are.
```suggestion
On the :guilabel:`Reporting` page (accessible via the :menuselection:`Reporting` option in the
header menu), there are options to apply different combinations of :guilabel:`Filters` and
:guilabel:`Measures` to view metrics in a number of different layouts, as in the :guilabel:`Graph`,
:guilabel:`List`, and :guilabel:`Cohort` views.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+Each :guilabel:`Reporting` metric view option allows for more extensive performance analysis of SMS
+mailings.
+
+For example, while in the default :guilabel:`Graph` view, SMS data is visualized as different graphs
+and charts, which can be sorted and grouped in various ways (e.g. :guilabel:`Measures` drop down
+menu).
+
+.. image:: sms_essentials/sms-reporting-page.png
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:158]: The crop is pretty good here, however this image is too wide @ 1920 pixels. Consider taking a shot that has more data, as well, instead of what we get by default using runbot. Images that show the depth of capabilities, I think, are more valuable to the reader and justifies adding the extra file weight to the page.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

Nice work. I like the depth of explanation and walk-through on these 👍 

My CRs below are mostly around small things and fine tuning since the content already hits the mark. See below where I made comments just on the `sms_essentials.rst` doc,   which apply to all docs in this PR, as well. 

Overview:
- formatting of sentences, mainly parenthesis use. I see what you're doing in most cases, using parenthesis as tiny _aside_ comments in sentences, however, I don't think they're needed in most cases. Most of what I see comes off as redundant or could easily be included into the sentence with a comma.
- line breaks. Just a few spots to fix.
- headings. Two headings are duplicates and we don't need `How to...` prefixes.
- images. Most images are too large, not in HD, and could use red markup to illustrate cause > effect. As well, some screenshots would benefit from showcasing more information or a more complete use case to illustrate the depth of the product.

Can you take a look and apply to every doc on the PR? Can work with @samueljlieber to push up the next batch of changes.

Thank you both!

**Comment by samueljlieber** [comment]
Hi @ksc-odoo I implemented your changes in https://github.com/odoo/documentation/commit/ac8b1508a281defa2f2bfef8e010e14fab573d61 🙂

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+.. note::
+   Don't forget to hit the :guilabel:`Save` button once changes have been made.
+
+Mailing lists contacts
+======================
+
+If :guilabel:`Mailing List Contacts` is selected from the :guilabel:`Mailing Lists` header drop-down
+menu, Odoo reveals a separate page with all the contacts associated with one (or more) of the
+configured mailing lists in the database.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Rewording to remove passive/conditional tense in favor of active.
```suggestion
Access contacts information from one or more mailing lists navigate to :menuselection:`Mailing Lists
--> Mailing List Contacts` to reveal a dashboard with with all the contacts associated with one or
more of the configured mailing lists in the database.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+.. note::
+   By default, Odoo reveals the :guilabel:`Mailing List Contacts` page with the :guilabel:`Exclude
+   Blacklisted Phone` filter in the search bar. Therefore only showing contact information for
+   recipients who still want to receive communications and mailings.
+
+Communication history in the Chatter
+------------------------------------
+
+An accessible record of the sent mailings is kept on each recipient's :guilabel:`Chatter`, located
+beneath a recipient's contact form (in the :guilabel:`Contacts` application).
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Don't think these get guilabels since it's not active instruction on the UI.
```suggestion
An accessible record of every sent mailing(s) is kept on each recipient's *chatter* section, located
beneath a recipient's contact form (in the *Contacts* application).
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+Database users can reference the :guilabel:`Chatter` to easily keep track of communications, and see
+a history of interactions with contacts and prospects.
+
+For example, sales representatives can use the :guilabel:`Chatter` to quickly find out which SMS
+mailing promotions a certain customer has received (or not received).
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Same here.
When referencing BE docs they don't use guilabels or capitalize this term.
We can mirror that style, for now, and can clarify at some point.
```suggestion
Database users can reference the chatter to easily keep track of communications, and see a history
of interactions with contacts and prospects.

For example, sales representatives can use the chatter to quickly find out which :abbr:`SMS (Short
Message Service)` mailing promotions a certain customer has received (or not received).
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+.. image:: mailing_lists_blacklists/sms-marketing-chatter.png
+   :align: center
+   :alt: View of what the chatter looks like in the Odoo Contacts app.
+
+Blacklist
+=========
+
+:guilabel:`Odoo SMS Marketing` has a :guilabel:`Blacklist` feature that provides recipients with the
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: No guilabel here
```suggestion
Odoo *SMS Marketing* has a :guilabel:`Blacklist` feature that provides recipients with the power to
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+power to add their phone number to a list of people who no longer want to receieve communications or
+mailings.
+
+This is also known as the "Unsubscribe Process." Customers will automatically be added onto the
+:guilabel:`Blacklist`, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription
+Management Page`. Customers can also be added manually to the blacklist, if necessary.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Formatting
```suggestion
This is also known as the unsubscribe process: customers will automatically be added onto a
*blacklist*, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription Management`
page. Customers can also be added manually to the blacklist, if necessary.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+
+To see a complete collection of blacklisted numbers, navigate to the :guilabel:`SMS Marketing` app,
+and go to :menuselection:`Configuration --> Blacklisted Phone Numbers`. Doing so, reveals a separate
+page containing every blacklisted phone number in the database.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Wording/formatting.
```suggestion
To see a complete collection of blacklisted numbers, navigate to the :menuselection:`SMS Marketing
app --> Configuration --> Blacklisted Phone Numbers` to reveal a dashboard containing every
blacklisted phone number in the database.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+
+.. image:: mailing_lists_blacklists/sms-blacklist-menu.png
+   :align: center
+   :alt: SMS Blacklist menu in the application.
+
+To manually add a number, click the :guilabel:`Create` button in the upper-left. When clicked, Odoo
+reveals a separate page, in which the phone number to be blacklisted is entered. There's also a
+checkbox to indicate whether that particular phone numnber is :guilabel:`Active` (or not).
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: "Odoo reveals a separate page" phrase is too wordy. Just say what's happening in clear terms 😉 
```suggestion
To manually add a number to a blacklist, click the :guilabel:`Create` button in the upper-left
corner of the dashboard and enter the phone number on the next page's form. There's also a
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+
+.. image:: mailing_lists_blacklists/sms-blacklist-create.png
+   :align: center
+   :alt: SMS Blacklist menu in the application.
+
+Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklist`. To
+remove any number from the :guilabel:`Blacklist`, select the desired number, and click
+:guilabel:`Unblacklist`.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: `Blacklisted Phone Numbers` gets a guilabel because it's a menu item and the name of a dashboard, while _blacklist_ does not. Added some more specificity here too.

```suggestion
Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklisted Phone
Numbers` list. To remove any number from the blacklist, select the desired number on the dashboard,
and then, on the phone number's form, click :guilabel:`Unblacklist`.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+
+Importing blacklists
+--------------------
+
+During a software/platform migration, it is possible to import an already existing blacklist of
+contacts. This would include customers, who have already asked to be :guilabel:`Blacklisted` on SMS
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]:
```suggestion
contacts. This would include customers, who have already asked to be blacklisted` on :abbr:`SMS
(Short Message Service)` mailings.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+mailings.
+
+To do that, navigate to the :guilabel:`SMS Marketing` app, and go to :menuselection:`Configuration
+--> Blacklisted Phone Numbers`. Then, select the :guilabel:`Favorites` drop-down (beneath
+the search bar), and click :guilabel:`Import records`.
```
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]:
```suggestion
To do that, navigate to :menuselection:`SMS Marketing app --> Configuration --> Blacklisted Phone
Numbers`, and then select the :guilabel:`Favorites` drop-down menu (beneath the search bar), and
click :guilabel:`Import records`.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 👋 

A few more change requests here. See below and on [01d323d](https://github.com/odoo/documentation/pull/2553/commits/01d323d8b8a0883aeee72e3c1bb164683bb79d9d). 

If you agree with the changes, let me know so I can squash/merge, otherwise indicate if and where you might want to go over something in more detail.

Ty/lmk 🤙

**Review by StraubCreative — CHANGES_REQUESTED** [review]
EDIT: see [review notes above](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1358117349). GH bugged out and submitted the review twice 😓

**Comment by ksc-odoo** [comment]
> EDIT: see [review notes above](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1358117349). GH bugged out and submitted the review twice 😓



> If you agree with the changes, let me know so I can squash/merge, otherwise indicate if and where you might want to go over something in more detail.

Hey @StraubCreative I agree with ALL these suggested changes - feel free to squash/merge at your earliest convenience. Thanks!

**Comment by StraubCreative** [comment]
@robodoo r+

**Review by StraubCreative — APPROVED** [review]
